README.md

```diff
@@ -90,26 +92,30 @@ func handler() {
 		"key3":   "val3",
 	}
 
-	// Register Counter with desired tags.
-	customRawCounter := metrics.NewCounter()
-	wavefront.RegisterMetric("counter", customRawCounter, appTags)
-	customRawCounter.Inc(1)
-
-	// Register Delta Counter with desired tags.
-	customDeltaCounter := metrics.NewCounter()
-	deltaCounterName := wavefront.DeltaCounterName("deltaCounter")
-	wavefront.RegisterMetric(deltaCounterName, customDeltaCounter, appTags)
-	customDeltaCounter.Inc(1)
-
-	// Register Gauge with desired tags.
-	gaugeValue := metrics.NewGauge()
-	wavefront.RegisterMetric("gaugeValue", gaugeValue, appTags)
-	gaugeValue.Update(551)
-
-	// Register Float Gauge with desired tags.
-	gaugeFloatValue := metrics.NewGaugeFloat64()
-	wavefront.RegisterMetric("gaugeFloatValue", gaugeFloatValue, appTags)
-	gaugeFloatValue.Update(551.4)
+	dc := &wavefront.DirectConfiguration{
```

Comment:
I missed this earlier, but realized when I saw your modified example in the README. 
If user sent a custom metric, then we appended the metric level appTags and the standard lambda point tags (aws region, aws function name etc.) to his custom metric. We need to support that with this change as well or existing functionality is broken. We could achieve this easily since go-metrics-wavefront supported reporter level tags earlier.  will let you know how to proceed on this.